actionpack/lib/action_controller.rb

```diff
@@ -54,6 +54,12 @@ module ActionController
   autoload :TemplateAssertions, "action_controller/test_case"
 end
 
+ActiveSupport.on_load(:after_initialize) do
+  if config.eager_load
+    ActionController::Base.descendants.each(&:action_methods)
```

Comment:
@matthewd `AbstractController::Base.descendants` would also include ActionMailer, and I thought we'd want to isolate eager loading controller actions and mailer actions.

With the current state of the PR, ActionMailer actions would be eager loaded from a separate ActionMailer-specific callback.

Do you think it makes sense to use only one callback to eager load actions from both ActionController and ActionMailer?